packages/flutter/lib/src/material/popup_menu_theme.dart

```diff
@@ -0,0 +1,121 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show lerpDouble;
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+
+/// Defines default property values for [PopupMenuEntry]'s [Material].
+///
+/// Descendant widgets obtain the current [PopupMenuEntryThemeData] object
+/// using `Theme.of(context).popupMenuEntryTheme`. Instances of
+/// [PopupMenuEntryThemeData] can be customized with
+/// [PopupMenuEntryThemeData.copyWith].
+///
+/// Typically a [PopupMenuEntryThemeData] is specified as part of the
+/// overall [Theme] with [ThemeData.popupMenuEntryThemeData].
+///
+/// All [PopupMenuEntryThemeData] properties are `null` by default.
+/// When null, the [PopupMenuEntry] will provide its own defaults.
+///
+/// See also:
+///
+///  * [ThemeData], which describes the overall theme information for the
+///    application.
+class PopupMenuEntryThemeData extends Diagnosticable {
+  /// Creates a theme that can be used for [ThemeData.popupMenuEntryTheme].
+  const PopupMenuEntryThemeData({
+    this.surfaceContainerColor,
+    this.shape,
+    this.elevation,
+    this.textStyle,
+  });
+
+  /// Default value for [_PopupMenuRoute.color].
+  final Color surfaceContainerColor;
```

Comment:
This is a bit of a mouthful, `color` would be enough (and consistent with other themes) I think

Reply:
Done